scss/civicrm/contact/pages/_export-contact.scss

```diff
@@ -36,10 +44,30 @@
     }
   }
 
-  .crm-submit-buttons {
-    > .crm-button {
-      margin-left: 0 !important;
-      margin-right: 10px !important;
+  .crm-form-block {
+    background-color: transparent;
+    box-shadow: none;
+
+    > .crm-submit-buttons {
+      background-color: #ffffff;
+      box-shadow: 0 8px 18px 0px rgba(0, 0, 0, .1);
+      margin: 0 !important;
```

Comment:
What this much importants? can it be handled by adding more selectors or making style more specific?